src/main/java/com/epam/reportportal/junit5/SystemAttributesFetcher.java

```diff
@@ -0,0 +1,56 @@
+package com.epam.reportportal.junit5;
+
+import com.epam.ta.reportportal.ws.model.attribute.ItemAttributesRQ;
+import org.slf4j.LoggerFactory;
+import rp.com.google.common.collect.Sets;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+import java.util.Set;
+
+public class SystemAttributesFetcher {
```

Comment:
Please add javadoc.
And unit-tests for this class.

Reply:
there are no public methods

Reply:
Doesn't mean that there is no bugs.
Also you call `static Set<ItemAttributesRQ> collectSystemAttributes(Boolean skippedAnIssue)` which is package-public and allows you to implement a test in the same package.

Reply:
done